lms/templates/calculator/toggle_calculator.html

```diff
@@ -17,121 +17,7 @@
-                      ${_("|| (parallel resistors)")}
-                    </td>
-                    <td>x+(2*y)/x-1
-                      x^(n+1)<br />
-                      v_IN+v_OUT<br />
-                      1||2
-                    </td>
-                  </tr>
-                  <tr>
-                    <th scope="row">${_("Greek letters")}</th>
-                    <td>${_("Name of letter")}</td>
-                    <td>alpha<br />
-                      lambda
-                    </td>
-                  </tr>
-                  <tr>
-                    <th scope="row">${_("Constants")}</th>
-                    <td>c, e, g, i, j, k, pi, q, T</td>
-                    <td>20*c<br />
-                      418*T
-                    </td>
-                  </tr>
-                  <tr>
-                    <th scope="row">${_("Affixes")}</th>
-                    <td>${_("Percent sign (%) and metric affixes (d, c, m, u, n, p, k, M, G, T)")}</td>
-                    <td>20%<br />
-                      20c<br />
-                      418T
-                    </td>
-                  </tr>
-                  <tr>
-                    <th scope="row">${_("Basic functions")}</th>
-                    <td>abs, exp, fact or factorial, ln, log2, log10, sqrt</td>
-                    <td>abs(x+y)<br />
-                      sqrt(x^2-y)
-                    </td>
-                  </tr>
-                  <tr>
-                    <th scope="row">${_("Trigonometric functions")}</th>
-                    <td>sin, cos, tan, sec, csc, cot<br />
-                      arcsin, sinh, arcsinh, etc.<br />
-                    </td>
-                    <td>sin(4x+y)<br />
-                      arccsch(4x+y)
-                    </td>
-                    <td></td>
-                  </tr>
-                  <tr>
-                    ## Translators: Please see http://en.wikipedia.org/wiki/Scientific_notation
-                    <th scope="row">${_("Scientific notation")}</th>
-                    ## Translators: 10^ is a mathematical symbol. Please do not translate.
-                    <td>${_("10^ and the exponent")}</td>
-                    <td>10^-9</td>
-                  </tr>
-                  <tr>
-                    ## Translators: this is part of scientific notation. Please see http://en.wikipedia.org/wiki/Scientific_notation#E_notation
-                    <th scope="row">${_("e notation")}</th>
-                    ## Translators: 1e is a mathematical symbol. Please do not translate.
-                    <td>${_("1e and the exponent")}</td>
-                    <td>1e-9</td>
-                  </tr>
-                </tbody>
-              </table>
-            </li>
-          </ul>
+          <a href="#" id="calculator_hint" aria-haspopup="true" aria-expanded="false" aria-controls="calculator_input_help" aria-describedby="hint-instructions"><span class="calc-hint sr">${_("Hints")}</span></a>
```

Comment:
@OmarIthawi in the previous design, pressing the Hints button revealed the sibling container so that the tab order remained logical.  In this design, the container is no longer a sibling, which breaks the logical tab order.  You will have to manually manage focus so that it remains logical.